tailcfg/tailcfg.go

```diff
@@ -1754,26 +1755,35 @@ type Debug struct {
 	// SleepSeconds requests that the client sleep for the
 	// provided number of seconds.
 	// The client can (and should) limit the value (such as 5
-	// minutes).
+	// minutes). This exists as a safety measure to slow down
+	// spinning clients, in case we introduce a bug in the
+	// state machine.
 	SleepSeconds float64 `json:",omitempty"`
 
-	// RandomizeClientPort is whether magicsock should UDP bind to
-	// :0 to get a random local port, ignoring any configured
-	// fixed port.
+	// RandomizeClientPort is whether magicsock should UDP bind to :0 to get a
+	// random local port, ignoring any configured fixed port.
 	//
-	// Deprecated: use NodeAttrRandomizeClientPort instead.
+	// Deprecated: use NodeAttrRandomizeClientPort instead. This is kept in code
+	// only so the control plane can use it to send to old clients.
 	RandomizeClientPort bool `json:",omitempty"`
 
-	// OneCGNATRoute controls whether the client should prefer to make one
-	// big CGNAT /10 route rather than a /32 per peer.
+	// OneCGNATRoute controls whether the client should prefer to make one big
+	// CGNAT /10 route rather than a /32 per peer.
+	//
+	// Deprecated: use NodeAttrOneCGNATEnable or NodeAttrOneCGNATDisable
+	// instead. This is kept in code only so the control plane can use it to
+	// send to old clients.
 	OneCGNATRoute opt.Bool `json:",omitempty"`
 
 	// DisableLogTail disables the logtail package. Once disabled it can't be
 	// re-enabled for the lifetime of the process.
+	//
+	// This is primarily used by Headscale.
 	DisableLogTail bool `json:",omitempty"`
 
 	// Exit optionally specifies that the client should os.Exit
-	// with this code.
+	// with this code. This is a safety measure in case a client is crash
+	// looping or in an unsafe state and we need to remotely shut it down.
 	Exit *int `json:",omitempty"`
```

Comment:
Could be c2n

Reply:
Could. But then if we ever need this there are conditionals in control by capver. We've never needed it. Just one scare IIRC.